webtau-groovy/src/test/groovy/com/twosigma/webtau/cfg/WebTauGroovyFileConfigHandlerTest.groovy

```diff
@@ -73,14 +73,43 @@ class WebTauGroovyFileConfigHandlerTest {
         } should throwException(IllegalArgumentException, ~/environment <unknown> is not defined in/)
     }
 
+    @Test
+    void "should capture nested config values"() {
+        def cfg = createNestedConfig()
+        cfg.envConfigValue.set('test', 'prod')
+        handle(cfg)
+
+        cfg.get('parent').should == [child: 'pc-1', another: 'pc-2']
+        cfg.parent.child.should == 'pc-1'
+
+        cfg.get('additionalUrls').should == [appOne: 'http://app-one-prod', appTwo: 'http://app-two-prod']
+    }
+
+    @Test
+    void "should allow to override a single value within a nested object for an env"() {
+        def cfg = createNestedConfig()
+        cfg.envConfigValue.set('test', 'dev')
+        handle(cfg)
+
+        cfg.get('additionalUrls').should == [appOne: 'http://app-one-dev', appTwo: 'http://app-two']
```

Comment:
sweet! to demonstrate usage of the new feature, you can demonstrate groovy path access to appOne as 
`cfg.get('additionalUrls.appOne')` 
That'd be a great sample code addition to the doc as well.

Reply:
... and as I scroll up, that's an action item on me :-)